cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -850,10 +850,31 @@ pCommitteeHotKey =
     , Opt.help "Constitutional Committee hot key (hex-encoded)."
     ]
 
-pCommitteeHotVerificationKey :: Parser (VerificationKey CommitteeHotKey)
-pCommitteeHotVerificationKey =
+pCommitteeHotVerificationKeyOrFile :: Parser (VerificationKeyOrFile CommitteeHotKey)
+pCommitteeHotVerificationKeyOrFile =
+  asum
+    [ VerificationKeyValue <$> pCommitteeHotVerificationKey "hot-verification-key"
+    , VerificationKeyFilePath <$> pCommitteeHotVerificationKeyFile "hot-verification-key-file"
+    ]
+
+pCommitteeHotVerificationKeyHash :: Parser (Hash CommitteeHotKey)
+pCommitteeHotVerificationKeyHash =
+  Opt.option (Opt.eitherReader deserialiseFromHex) $ mconcat
```

Comment:
I think `pHexHash` does that already

Reply:
Indeed, but I'd need a small refactor to reuse it; so created follow-up issue: https://github.com/IntersectMBO/cardano-cli/issues/811